src/bots/pins/index.js

```diff
@@ -0,0 +1,4 @@
+import client from '../../client';
+import handler from './handler';
+
+client.on('pins', (args) => handler(args));
```

Comment:
Should we create a healthcheck minibot?
Just literally a ping command for this sort of occasion.